admin_guide/backup_restore.adoc

```diff
@@ -387,7 +387,7 @@ and displays {product-title}'s configuration:
 +
 ----
 # systemctl daemon-reload
-# systemctl restart etcd
+# systemctl master-restart etcd
```

Comment:
systemctl is redundant, the command line running result is: 
# systemctl master-restart etcd
Unknown operation 'master-restart'.
so master-restart can't work with systemctl, the correct command line is:  master-restart etcd, similar for 'api' and 'controller' in other place in this doc.